### ISSUE TITLE
recording app: "record checked by" indication on unit and locus tabs

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#2397]
I am envisioning using Kiosk to input significant amounts of legacy data made by older projects. In the scenario I envision, which is based on the hypothetical case of Abydos, the records are voluminous and include at least two significant hurdles for turning into Kiosk data: the handwriting of the original records sucks, and the records were not kept on context sheets but rather in notebooks and so there is a level of interpretation even to turning things into form-based recording. (I would start with my own records, which show a split. The later ones, once I was at Brown, are on context forms that I designed that are in a way the origin of Kiosk itself. But the earlier ones are notebooks.)

This would necessarily be a team of people doing data entry and no single person would look at all the handwritten records, probably not even at all of the electronic records. So the process needs to be systematized and to include checks. One of those that I envision is that each electronic record is checked against the handwritten data by a second person. A team doing this would need both a way to indicate records that have been checked and by whom, and (probably just in text, but would need a systematic method) a way of indicating irresolvable discrepancies between the readings done by those two people. The second reader will catch outright errors, of course, but also things that they simply read differently. I can also imagine a use for a way of flagging things for a third, perhaps directorial, checker to come in and make an executive decision. 

This would be easy in CM - just make a to do/ done for it. But that's also the only thing Abydos already has in a FM database, the CM records. (Still needs checking against the notebooks; the records GOT in the database in a way that is not straightforward or error-proof, though registrars should have caught a lot of it.)

Units and Loci for sure would need to have a way of indicating this. A report producing a list of what is left to check would be useful. 

I am seriously considering using May and Heidi this winter to digitize my own old Abydos records, so this is not just hypothetical and would have a live test scenario of utility.

In the end I would need to be able to sort even the physical notebooks on this basis. At least the old practice was to have a notebook per unit. I need to be able to say "these notebooks in this pile have been entirely and definitively entered into Kiosk and checked and they can be relegated to off-site storage; this pile we still have questions about/ things to do with them."

**Comment by lbestock** [comment]
Ugh needs thinking about photos, too. That will be a nightmare.

**Comment by urapadmin** [comment]
we are doing it without this, so I think we can close this, right?

**Comment by lbestock** [comment]
Yes. In an ideal world we’d do it, but realistically it’s never going to happen.
